### PR TITLE
Separate create upgrade

### PR DESCRIPTION
**Pull request by ewolinetz**
This is to address https://bugzilla.redhat.com/show_bug.cgi?id=1664497

### CODE REVIEW

File: hack/cr.yaml [hunk 1]
```diff
@@ -6,15 +6,17 @@ metadata:
 spec:
   managementState: "Managed"
   nodeSpec:
-    image: openshift/origin-logging-elasticsearch5:latest
+    image: docker.io/openshift/origin-logging-elasticsearch5:latest
+    resources:
+      limits:
+        memory: 1Gi
```
Comment (jcantrill) [inline hack/cr.yaml]: Understand this is a 'move' but why are we not specifying the values to be the same given this is our general strategy in prior releases to ensure the node is properly scheduled.

Comment (ewolinetz) [reply hack/cr.yaml]: what do you mean when you're saying they aren't the same? Are you referring to the ansible defaults?

**Comment by richm** [comment]
lgtm too

**Pull request merged**